Header header
string sensor_id
uint64 frame_id
float64 sensor_timestamp
uint16 width
uint16 height
uint16 width_step
uint16 height_step
uint32[] fg_indices
uint32[] bg_fringe_indices
uint32[] indices
uint16[] depth
uint8[] color
#float32[] reserve_float
#int32[] reserve_int